js/adapt-stateful-session.js

```diff
@@ -179,15 +180,28 @@ export default class StatefulSession extends Backbone.Controller {
 
   initializeContentObjectives() {
     if (!this.shouldRecordObjectives) return;
-    Adapt.contentObjects.forEach(model => {
-      if (model.isTypeGroup('course')) return;
-      const id = model.get('_id');
-      const description = model.get('title') || model.get('displayTitle');
-      offlineStorage.set('objectiveDescription', id, description);
-      if (model.get('_isVisited')) return;
-      const completionStatus = COMPLETION_STATE.NOTATTEMPTED.asLowerCase;
-      offlineStorage.set('objectiveStatus', id, completionStatus);
-    });
+    Adapt.contentObjects.forEach(model => this.initializeContentObjective(model));
+  }
+
+  initializeContentObjective(model) {
```

Comment:
Does this need to be a separate function? Is it called from anywhere else?

Reply:
I think this was left over from an alternative implementation of how _isAvailable content objects were handled:
https://github.com/adaptlearning/adapt-contrib-spoor/commit/fdb9f1550207a866588b05b57d073e6bbed4d668

Happy to combine this now we only ever intialize them once.

Reply:
Seems odd having the parent caller be a one liner.